backend/climateconnect_api/models/common.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+
+class Availability(models.Model):
+    name = models.CharField(
+        help_text="Full text of user's availability to work on a project",
+        verbose_name="Name",
+        max_length=512
+    )
+
+    key = models.CharField(
+        help_text="Key for user' availability",
+        verbose_name="Key",
+        max_length=100
+    )
+
+    created_at = models.DateTimeField(
+        help_text="Time when object was created",
+        verbose_name="Created At",
+        auto_now_add=True
+    )
+
+    updated_at = models.DateTimeField(
+        help_text="Time when object was updated",
+        verbose_name="Updated At",
+        auto_now=True
+    )
+
+    class Meta:
+        app_label = "climateconnect_api"
+        verbose_name = "Availability"
+        verbose_name_plural = "Availabilities"
+        db_table = "climateconnect_availability"
+
+    def __str__(self):
+        return "%s" % self.name
+
+
+class Skill(models.Model):
+    name = models.CharField(
+        help_text="Name of the skill",
+        verbose_name="Name",
+        max_length=512
+    )
+
```

Comment:
Maybe it makes sense to additionally have a key or id field incase we want to change the name of a skill later?

Reply:
Djnago automatically creates a primary key in the postgres database.

Reply:
So no need to explicitly define it in model class